modules/ingest-geoip/src/test/java/org/elasticsearch/ingest/geoip/MaxMindSupportTests.java

```diff
@@ -156,9 +156,14 @@ public class MaxMindSupportTests extends ESTestCase {
     private static final Set<String> CONNECT_TYPE_SUPPORTED_FIELDS = Set.of("connectionType");
     private static final Set<String> CONNECT_TYPE_UNSUPPORTED_FIELDS = Set.of("ipAddress", "network");
 
-    private static final Set<String> COUNTRY_SUPPORTED_FIELDS = Set.of("continent.name", "country.isoCode", "country.name");
-    private static final Set<String> COUNTRY_UNSUPPORTED_FIELDS = Set.of(
+    private static final Set<String> COUNTRY_SUPPORTED_FIELDS = Set.of(
+        "continent.name",
+        "country.isoCode",
         "continent.code",
+        "country.name"
+    );
+    private static final Set<String> COUNTRY_UNSUPPORTED_FIELDS = Set.of(
+
```

Comment:
The tiniest nit: was this whitespace intended?

Reply:
No, gross! I'll fix it.

Reply:
6c5de75d4d8505c099210936f1a83886ebcd73f9